.gitignore

```diff
@@ -1 +1,3 @@
-node_modules
\ No newline at end of file
+node_modules,
```

Comment:
no comma at the end, you've gone from 10,000 lines to 710,000 :P

Reply:
Thanks for your help in fixing this and my apologies for the mess. 
The latest push didn't include those files, so it seems good. Thanks!